### PR TITLE
created html&css added linters. 

### PR DESCRIPTION
**Pull request by aakbarkhan**
created the first Html and gitignore, implemented CSS.

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (wandji20) [inline MIT.md:10]: - [ ] It will be nice to customize this license.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @aakbarkhan  
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for Hello Microverse.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark: All linter checks are passing.
:heavy_check_mark: Commit messages are descriptive.

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Submit for a re-review once you are ready!
- [x] Pull Request should have a summary describing introduced changes in more details

- [x] Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section') instead of microverse

- [x] Files that are irrelevant to run this project should be ignored (eg .hintrc, .stylelintrc.json, etc)

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```
Comment (DanteAlonsoHT) [inline MIT.md:1]: - [ ] [OPTIONAL] Instead of `[YOUR NAME]` you can add your name there.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello Microverse repository contains html and css.
+
+![screenshot](./hello-microverse.png)
+
+Hello Microverse project.
```
Comment (DanteAlonsoHT) [inline README.md:9]: - [ ] [OPTIONAL] For the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description from your README file for the next time 😉).

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @aakbarkhan,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**